cli/args/flags.rs

```diff
@@ -447,6 +447,7 @@ pub struct Flags {
   pub unstable: bool,
   pub unstable_bare_node_builtins: bool,
   pub unstable_byonm: bool,
+  pub unstable_loose_imports: bool,
```

Comment:
Alternative name to further discourage use: `--unstable-sloppy-imports`

Reply:
I also like `--unstable-sloppy-imports` better.

Reply:
Updated.